ccpp/data/GFS_typedefs.F90

```diff
@@ -3947,6 +3949,13 @@ subroutine control_initialize (Model, nlunit, fn_nml, me, master, &
           write(0,*) "Logic error, RRTMGP flag doGP_sgs_mynn only works with do_mynnedmf=.true."
           stop
        endif
+       if (Model%doGP_sgs_cnv .or. Model%doGP_sgs_mynn) then
+          write(0,*) "RRTMGP explicit cloud scheme being used."
+          Model%doGP_smearclds = .false.
+       else
+           write(0,*) "RRTMGP implicit cloud scheme being used."
```

Comment:
Why is this written to stderr, it is not error message. Could this be written as `write(*,*) " ....."` or just using print statement.

Reply:
I think write(*,*) would be the best choice. Since all regression tests have been done. we can leave it as is for now and modify it in a future PR.

Reply:
I agree. We should be able to add a quick fix in the next pr.